metricbeat/module/mysql/mysql.go

```diff
@@ -86,6 +114,10 @@ func ParseDSN(mod mb.Module, host string) (mb.HostData, error) {
 	noCredentialsConfig.User = ""
 	noCredentialsConfig.Passwd = ""
 
+	if c.TLS.IsEnabled() {
+		config.TLSConfig = TLSConfigKey
+	}
```

Comment:
why is this different?

Reply:
That's the TLS config name being appended to the mysql DSN.